content/en/about/faq/distributed-tracing/how-to-support-tracing.md

```diff
@@ -19,7 +19,7 @@ If you are using Lightstep, you will also need to forward the following headers:
 
 - `x-ot-span-context`
 
-If you are using OpenTelemetry, OpenCensus or Stackdriver, you will also need to forward the following headers:
+If you are using OpenTelemetry or Stackdriver, you will also need to forward the following headers:
```

Comment:
what about Stackdriver? I saw some PRs removing stackdriver as well?

Reply:
Good call.. let me double check

Reply:
otherwise LGTM

Reply:
I can only find tests being removed and comments from last week that it *will* be removed (future tense). So I think we should do Stackdriver removals when it is removed.

Reply:
sounds good

Reply:
fyi @kyessenov